packages/core/types/index.d.ts

```diff
@@ -2247,6 +2247,11 @@ export namespace Asciidoctor {
      */
     setSectionName(value: string): void;
 
+    /**
+     * Get the section numeral of this section.
+     */
+    getSectionNumeral(): string;
+
```

Comment:
Almost done, we also need to add `getSectionNumber` in the definition type.

Reply:
Ha believe it or not I did this and had not yet saved the file.